os-server/src/main/java/app/openschool/OpenSchoolApplication.java

```diff
@@ -20,6 +22,12 @@
                     + "get information about courses, their categories, and mentors, "
                     + "for authenticated mentors to create courses, "
                     + "and for administrators to register mentors"))
+@SecurityScheme (
```

Comment:
Could you please create a Config class for openAPI and add these annotations there?